tuna/connection.py

```diff
@@ -190,11 +277,12 @@ def exec_command(self, cmd, timeout=SSH_TIMEOUT, abort=None, proc_line=None):
       if not proc_line:
         # pylint: disable-next=unnecessary-lambda-assignment ; more readable
         proc_line = lambda x: self.logger.info(line.strip())
-      ret_out = StringIO()
-      ret_err = e_var
+      ret_out: Union[TextIO, StringIO] = StringIO()
+      ret_err: Union[ChannelStderrFile, StringIO] = e_var
```

Comment:
StringIO type should be sufficient.

Reply:
connection.py:281: error: Incompatible types in assignment (expression has type "ChannelStderrFile", variable has type "StringIO")
Found 1 error in 1 file (checked 1 source file)
Either we have to use #type :ignore or add ChannelStderrFile as possible datatype. because, e_var is ChannelStderrFile type.

Reply:
Looks to me like ret_err is set to a StringIO in the try. Looks like this e_var value is only used in case of an exception, so I think it may not be necessary to keep it.

Reply:
as above explained